libs/service-portal/vehicles/src/screens/VehicleDetail/VehicleDetail.tsx

```diff
@@ -0,0 +1,271 @@
+  Divider,
+  GridColumn,
+  GridRow,
+  Table as T,
+  Stack,
+  Text,
+  LoadingDots,
+} from '@island.is/island-ui/core'
+import {
+  ServicePortalModuleComponent,
+  UserInfoLine,
+} from '@island.is/service-portal/core'
+import { useLocale, useNamespaces } from '@island.is/localization'
+
+import BaseInfoItem from '../../components/DetailTable/BaseInfoItem'
+import RegistrationInfoItem from '../../components/DetailTable/RegistrationInfoItem'
+import OwnerInfoItem from '../../components/DetailTable/OwnerInfoItem'
+import InspectionInfoItem from '../../components/DetailTable/InspectionInfoItem'
+import TechnicalInfoItem from '../../components/DetailTable/TechnicalInfoItem'
+import OwnersTable from '../../components/DetailTable/OwnersTable'
+import { gql, useQuery } from '@apollo/client'
+import { CurrentOwnerInfo, Query } from '@island.is/api/schema'
+import OperatorInfoItem from '../../components/DetailTable/OperatorInfoItem'
+import CoOwnerInfoItem from '../../components/DetailTable/CoOwnerInfoItem'
+
+const GET_USERS_VEHICLE_DETAIL = gql`
+  query GetUsersVehicles($input: GetVehicleDetailInput!) {
+    getVehicleDetail(input: $input) {
+      mainInfo {
+        model
+        subModel
+        regno
+        year
+        co2
+        cubicCapacity
+        trailerWithBrakesWeight
+        trailerWithoutBrakesWeight
+      }
+      basicInfo {
+        model
+        regno
+        subModel
+        permno
+        verno
+        year
+        country
+        preregDateYear
+        formerCountry
+        importStatus
+      }
+      registrationInfo {
+        firstRegistrationDate
+        preRegistrationDate
+        newRegistrationDate
+        vehicleGroup
+        color
+        reggroup
+        passengers
+        useGroup
+        driversPassengers
+        standingPassengers
+      }
+      currentOwnerInfo {
+        owner
+        persidno
+        address
+        postalcode
+        city
+        dateOfPurchase
+      }
+      inspectionInfo {
+        type
+        date
+        result
+        plateStatus
+        nextInspectionDate
+        lastInspectionDate
+      }
+      technicalInfo {
+        engine
+        totalWeight
+        cubicCapacity
+        capacityWeight
+        length
+        vehicleWeight
+        width
+        trailerWithoutBrakesWeight
+        horsepower
+        trailerWithBrakesWeight
+        carryingCapacity
+        axleTotalWeight
+        axle {
+          axleMaxWeight
+          wheelAxle
+        }
+      }
+      ownersInfo {
+        name
+        address
+        dateOfPurchase
+      }
+      coOwners {
+        persidno
+        owner
+        address
+        postalcode
+        city
+        dateOfPurchase
+      }
+      operator {
+        persidno
+        name
+        address
+        postalcode
+        city
+        startDate
+        endDate
+      }
+    }
+  }
+`
+
+const VehicleDetail: ServicePortalModuleComponent = ({ userInfo }) => {
+  useNamespaces('sp.vehicle')
```

Comment:
specific namespace for `sp.vehicle` vs `sp.vehicles` ?

Reply:
✅